olive/cli/auto_opt.py

```diff
@@ -338,6 +342,9 @@ def _get_passes_config(self, config: Dict[str, Any], olive_config: OlivePackageC
             # model already comes in int4
             passes_to_remove.add("matmul4")
 
+        if self.args.use_model_builder or not self.args.use_ort_genai:
```

Comment:
this option also requires this to be set in the onnx conversion pass https://github.com/microsoft/Olive/blob/81037d7d03879c627988b42457d3a254f7803cc7/olive/cli/capture_onnx.py#L204